3D Chess/Assets/Scripts/PieceEntity.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Realms;
+using UnityEngine;
+
+public class PieceEntity : RealmObject
+{
+    public int Type { get; set; }
```

Comment:
We can make `Type` private and expose `PieceType` publicly.

Reply:
This also hides the two casts we have to make nicely in here. Changed with 6d79643.